datasafe-encryption/datasafe-encryption-impl/src/main/java/de/adorsys/datasafe/encrypiton/impl/pathencryption/PathEncryptor.java

```diff
@@ -0,0 +1,22 @@
+package de.adorsys.datasafe.encrypiton.impl.pathencryption;
+
+import de.adorsys.datasafe.encrypiton.api.types.keystore.SecretKeyIDWithKey;
+
+/**
+ * Path encryption cipher configurer.
+ */
+public interface PathEncryptor {
+
+    /**
+     * @param secretKeyEntry Use this secret key
+     * @return Path encryption cipher that uses {@code secretKey}
+     */
+    byte[] encrypt(SecretKeyIDWithKey secretKeyEntry, byte[] rawData);
```

Comment:
It can return and consume string, because it is specialized class - for Path only